Makefile

```diff
@@ -72,7 +72,7 @@ CRANE := $(BIN_DIR)/crane
 # End vendored tools
 
 # golangci-lint is GPL, so it must not be vendored.
-GOLANGCI_LINT_VERSION := v1.63.4
+GOLANGCI_LINT_VERSION := v2.0.2
```

Comment:
Can you pull out the golint update to a separate change?

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1635